src/foam/u2/tag/Input.js

```diff
@@ -48,7 +49,8 @@ foam.CLASS({
 
   methods: [
     function initE() {
-      if ( this.type ) this.attrs({type: this.type});
+      if ( this.type ) this.attrs({ type: this.type });
+      if ( this.list ) this.attrs({ list: this.list$ });
```

Comment:
Does this need to be reflected into a property?  Or could we just use .attrs()/.setAttribute() from the outside?